kratos/utilities/delaunator_utilities.h

```diff
@@ -0,0 +1,73 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+#if !defined(KRATOS_DELAUNATOR_UTILITIES)
+#define KRATOS_DELAUNATOR_UTILITIES
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/model_part.h"
```

Comment:
just for the record (i.e. leave it as is)
this would IMO be an appropriate use for forward declaration, since then this heavy header could be moved to the cpp

Reply:
OK